.github/workflows/release.yml

```diff
@@ -46,18 +46,46 @@ jobs:
           file: './system.json'
           field: 'download'
           value: "https://github.com/${{github.repository}}/releases/download/${{steps.tag.outputs.tag}}/${{steps.systemID.outputs.prop}}.zip"
+
+      - name: Get mininum
+        id: minimum
+        uses: notiz-dev/github-action-json-property@release
+        with: 
+          path: 'system.json'
+          prop_path: 'compatibility.minimum'
+
+      - name: Get verified
+        id: verified
+        uses: notiz-dev/github-action-json-property@release
+        with: 
+          path: 'system.json'
+          prop_path: 'compatibility.verified'
+
+      - name: Get maximum
+        id: maximum
+        uses: notiz-dev/github-action-json-property@release
+        with: 
+          path: 'system.json'
+          prop_path: 'compatibility.maximum'
+
+      - name: Foundry Release API
+        uses: fjogeleit/http-request-action@v1
+        with:
+          url: 'https://api.foundryvtt.com/_api/packages/release_version'
+          method: 'POST'
+          customHeaders: '{"Content-Type": "application/json", "Authorization" : "${{ secrets.FOUNDRY_KEY }}"}'
+          data: '{"dry_run": false, "id" : "${{steps.systemID.outputs.prop}}", "release": {"version" :  "${{steps.tag.outputs.tag}}", "manifest": "https://raw.githubusercontent.com/${{github.repository}}/${{steps.tag.outputs.tag}}/system.json", "notes" : "https://github.com/${{github.repository}}/releases/tag/${{steps.tag.outputs.tag}}", "compatibility" : {"minimum": "${{steps.minimum.outputs.prop}}", "verified": "${{steps.verified.outputs.prop}}", "maximum": "${{steps.maximum.outputs.prop}}"} } }'
           
-      - name: Generate Changelog
-        id: changelog
-        uses: mikepenz/release-changelog-builder-action@v2
-        env:
-          GITHUB_TOKEN: ${{ secrets.PAT }}
+      # - name: Generate Changelog
+      #   id: changelog
+      #   uses: mikepenz/release-changelog-builder-action@v2
+      #   env:
+      #     GITHUB_TOKEN: ${{ secrets.PAT }}
```

Comment:
I'm trying to figure out why these changes are showing (as well as others) when I've merged master into `effect-refactor` already.

Reply:
Not sure if this had something to do with it? 
![image](https://github.com/moo-man/WFRP4e-FoundryVTT/assets/11304207/4dfb60f7-b288-4a3f-bd6e-7a2ad2bbf973)

Because after my last commit I didn't see there changes from other commits 🤔

Reply:
I went ahead and merged current `effect-refactor` into my branch, which seem to fix the `files changed` tab.

Reply:
I switched the branches because that has fixed the issue before. But it looks good now.